Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Controller.mo

```diff
@@ -0,0 +1,2260 @@
+    final quantity="PressureDifference") if have_varPriPum and have_locDPRegPri
+    "Measured differential pressure between hot water supply and return in primary circuit"
+    annotation (Placement(transformation(extent={{-440,-290},{-400,-250}}),
+        iconTransformation(extent={{-140,-130},{-100,-90}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput dpHotWatSec_loc(
+    final unit="Pa",
+    displayUnit="Pa",
+    final quantity="PressureDifference") if not have_priOnl and
+    have_varSecPum and have_locDPRegSec
+    "Measured differential pressure between hot water supply and return in secondary circuit"
+    annotation (Placement(transformation(extent={{-440,-330},{-400,-290}}),
+        iconTransformation(extent={{-140,-160},{-100,-120}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uHotWatIsoVal[nBoi](
+    final unit=fill("1",nBoi),
+    displayUnit=fill("1",nBoi)) if have_heaPriPum
+    "Measured boiler isolation valve position signals from plant"
+    annotation (Placement(transformation(extent={{-440,-500},{-400,-460}}),
+      iconTransformation(extent={{-140,-280},{-100,-240}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uBypValPos(
+    final unit="1",
+    displayUnit="1") if have_priOnl
+    "Measured bypass valve position signal from plant"
+    annotation (Placement(transformation(extent={{-440,-540},{-400,-500}}),
+      iconTransformation(extent={{-140,-310},{-100,-270}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uPriPumSpe[nPumPri](
+    final unit=fill("1",nBoi),
+    displayUnit=fill("1",nBoi)) if not have_priOnl
+    "Measured primary pump speed signal from plant"
+    annotation (Placement(transformation(extent={{-440,-580},{-400,-540}}),
+      iconTransformation(extent={{-140,-340},{-100,-300}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yBoi[nBoi]
+    "Boiler status vector"
+    annotation (Placement(transformation(extent={{400,180},{440,220}}),
+      iconTransformation(extent={{100,80},{140,120}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yPriPum[nPumPri]
+    "Primary pump enable status vector"
+    annotation (Placement(transformation(extent={{400,-150},{440,-110}}),
+      iconTransformation(extent={{100,-80},{140,-40}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySecPum[nPumSec] if not have_priOnl
+    "Secondary pump enable status vector"
+    annotation (Placement(transformation(extent={{400,-360},{440,-320}}),
+      iconTransformation(extent={{100,-160},{140,-120}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yPla
+    "Plant enable signal"
+    annotation (Placement(transformation(extent={{400,280},{440,320}}),
+      iconTransformation(extent={{100,160},{140,200}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput TPlaHotWatSupSet(
+    final unit="K",
+    displayUnit="K",
+    final quantity="ThermodynamicTemperature")
+    "Plant hot water supply temperature setpoint"
```

Comment:
- Not an output point per G36 4.11. Should be removed.
- (Temperatures should rather have `displayUnit="degC".)